src/UI.ts

```diff
@@ -61,6 +62,7 @@ class UI {
     this.onVideoElTimeupdate = noop
     this.onVideoElVolumeChange = noop
     this.onLoadedMetaData = noop
+    this.onVolumeChange = noop
```

Comment:
The naming confuses me. Why are there two functions called
`onVideoElVolumeChange` and `onVolumeChange`? Maybe add `ima` to the name somehow if it only relates to that